sdks/python/apache_beam/io/gcp/datastore/v1new/datastoreio.py

```diff
@@ -26,20 +26,27 @@
 import logging
 import time
 
+from google.api_core.exceptions import ClientError
```

Comment:
Please import this in a try block as shown here

https://github.com/apache/beam/blob/39cf3fccbb3950b9fd0c8938301d192c3ca85cce/sdks/python/apache_beam/io/gcp/bigquery_tools.py#L68

Reply:
Done, but I'm not sure what to do in the except https://github.com/apache/beam/blob/bd4f5c773b03d5bb8b5837335cf590366f2b1b36/sdks/python/apache_beam/io/gcp/datastore/v1new/datastoreio.py#L52

Reply:
Nothing is needed in the except block. This is needed due to the way this dep is loaded, its not always available when the module is loaded in some cases (I am not sure why in detail though, some other devs know the details on this)

Reply:
we allow users to install Google Cloud dependencies optionally (thus the `pip install apache-beam[gcp]`). The way in which we support this is with the try/catch imports that we have here : )